docker-compose.enterprise.yml

```
@@ -55,8 +40,8 @@ services:
             USERADM_HAVE_ADDONS: 1
 
     mender-api-gateway:
-        environment:
-            HAVE_MULTITENANT: 1
+      volumes:
+        - ./config/traefik/traefik.enterprise.yaml:/etc/traefik/config/traefik.enterprise.yaml
```

for consistency maybe:
```suggestion
        - ./config/traefik/traefik.enterprise.yaml:/etc/traefik/config/traefik.enterprise.yaml:ro
```